### ISSUE TITLE
Marathon version - update needed, Dockerfile missing

### ISSUE THREAD

**Issue by VAdamec** [rancher/community-catalog#392]
As we miss Dockerfile (https://hub.docker.com/r/rancher/marathon/ contain just images) can you please update Marathon to latest stable version (1.3.5) ?

**Comment by pvcon13** [comment]
yeah, that would be great. Let me know how I can help with this.  Trying to get Vamp to work inside on older version of marathon is tricky. Rawmind has built an experimental catalog for Mesos DNS etc. Would like to get all these working together.

**Comment by pvcon13** [comment]
I am having issues with Marathon 1.1.1 accessing private docker hub repos with the methods outlined by marathon. I suspect because the rancher catalog for marathon is downrev. I really want to help get that upgraded. How can I actually help to get this catalog upgraded. Its becoming a blocker for me. Thanks.

**Comment by pvcon13** [comment]
Is this on hold?
